lib/FieldType/FieldTypeFormMapperDispatcher.php

```diff
@@ -25,10 +25,20 @@ class FieldTypeFormMapperDispatcher implements FieldTypeFormMapperDispatcherInte
     /**
      * FieldType form mappers, indexed by FieldType identifier.
      *
-     * @var FieldTypeFormMapperInterface[]
+     * @var FieldTypeFormMapperInterface[]|FieldValueFormMapperInterface[]|FieldDefinitionFormMapperInterface[]
```

Comment:
Because `FieldTypeFormMapperInterface` is deprecated, split into `FieldValue` and `FieldDefinition`. Without it, the value part isn't known, and PhpStorm / IDEs report errors on usage of the value related methods.